packages/loader/container-loader/src/loader.ts

```diff
@@ -262,6 +262,10 @@ export interface ILoaderServices {
  */
 export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" | "readBlob"> & {
     size: number;
+    /**
+     * Return an array of all blob IDs present in storage
+     */
+    getIds(): string[];
```

Comment:
Maybe getBlobIds()? Interface may include more things in the future, so it's useful for method names to be more descriptive